### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
This is a microverse project where I was required to create a single web page.
* Add index.html file
* Add *.css file
* Add README file
* Add linters for HTML and CSS
* Add .gitignore file
* Add .github/workflows

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      "eslint": "^7.32.0",
+      "eslint-config-airbnb-base": "^14.2.1",
+      "eslint-plugin-import": "^2.23.4",
+      "hint": "^6.1.4",
+      "stylelint": "^13.13.1",
+      "stylelint-config-standard": "^21.0.0",
+      "stylelint-csstree-validator": "^1.9.0",
+      "stylelint-scss": "^3.19.0"
+    }
+  }
```
Comment (emmanuellekamwa) [inline package.json]: Package.json and package-lock.json are not needed for the linters of this project. Please delete those files.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑**
Hello @hamid-murambiwa 👋🏽
You have done a great job on this project, but there are some more changes to be made to approve this project.

## Required Changes ♻️
- You did not add the title on your pull request. Please do that.
_Check the comments under the review._ 👇🏽👀

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+    *First microverse project submission.*
+
+![an image of the Hello World webpage](https://github.com/hamid-murambiwa/README-IMG/blob/main/Screenshot%202021-08-10%20at%2015.42.46.png)
+
+
+<h2>Built With</h2>
+*HTML 
+*CSS3
+
+<h2>Authors</h2>
```
Comment (DanteAlonsoHT) [inline README.md:11]: I strongly recommend you additional instructions about how to set up your repository in a local environment, for instance:

### Install

To get a local copy up and running follow these simple example steps.
- Open terminal
- Clone this project by the command `git clone URL/repository_name.git`
- `cd repository_name` folder
- Open `index.html` in your local browser or using Live Server in Visual Studio Code.

- [ ] [OPTIONAL] Add more instructions about your project.

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @hamid-murambiwa, 

Your project is complete!  Feel free to merge it :shipit: 
Congratulations, I know you did a good job! 🎉 

### Optional Required Changes:

**See the Optional Required Changes in the comments below** ⬇️ 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

<hr>
Happy coding, you have good potential as a software developer!👏👏😉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**